README.md

```diff
@@ -100,7 +100,7 @@ When you define your workflow in the `.github/workflows` directory of your repos
 
 | Input     | Description |
 |-----------|-------------|
-| `release` | <p>(Optional) MATLAB release to set up. You can specify R2021a or a later release. By default, the value of `release` is `latest`. If you do not specify `release`, the action sets up the latest release of MATLAB.<p/><p>**Example**: `release: R2023a`<br/>**Example**: `release: latest`</p>
+| `release` | <p>(Optional) MATLAB release to set up. You can specify R2021a or a later release. By default, the value of `release` is `latest`, which corresponds to the latest release of MATLAB.<p/><p>**Example**: `release: R2023a`<br/>**Example**: `release: latest`</p>
```

Comment:
Not sure how confusing the  `release: latest` example is. Someone might think that they have to specify it even though not necessary?

Reply:
I don't think it is too confusing. Even if folks specify it manually, it does no harm.